### DISCUSSION TITLE
Github on Windows

### DISCUSSION THREAD

**Discussion by ashish-cardiff** [community/community#55018] [New to GitHub]
I want to Use Github for Windows. My Requirement is such to create a Repository on Local Intranet Windows Machine that will be accessible to All Connected windows machine.
Can this be done with the free version of Git ?

**Comment by ichenglin** [comment]
I believe there's some confusion here as you specifically mentioned **["free version of Git"](https://github.com/orgs/community/discussions/55018#discussion-5180082)**.
**[Git](https://git-scm.com/)** is a **FREE** and open-source distributed version control system, in which there's **no paid options or plans** available.

I'm assuming that you **meant GitHub** (with paid plans), then in this situation you're not hosting the repository **locally**,
but instead **remotely** on Github's Git servers (even if you made its visibility to private, it's still hosted remotely).

If you wish to host it **locally**, you will have to set up your own **[Git servers](https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server)**.

**Reply by ashish-cardiff** [reply]
Yes, Apologies for the Mix-up of GIT and GITHUB,

Also,
I want to Host the Repository Locally and not remotely on Github's Git servers (even if you made its visibility to private, it's still hosted remotely) - which is what I am trying to avoid.
any Ideas on this? If any Doc or any reference for this might help a lot

**Reply by ichenglin** [reply]
You will have to set up your own **[Git servers](https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server)** to host repositories **locally**,
you could also check out **[this video](https://www.youtube.com/watch?v=ju9loeXNVW0)** for a walk-through on the installation.

**Comment by airtower-luna** [comment]
There are a lot of ways to host a Git repository locally:

* The simplest one is to use some kind of shared storage, and put a bare repository there (e.g. a directory that everyone can access over SSH). If you just need to share within a small team that might be enough.
* [Chapter 4 of the Git Book](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols) describes how to set up an HTTP(S) server with basic auth, or an unauthenticate Git protocol server.
* [Gitolite](https://gitolite.com/gitolite/) provides something similar, but with SSH access instead (I've set it up years ago at university).
* If you want something more fancy, there are web applications you can host yourself, like [Gitea](https://gitea.io/en-us/) (I haven't hosted it myself, but know people who do).
